db/schema.rb

```diff
@@ -10,7 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_24_004600) do
+ActiveRecord::Schema[7.0].define(version: 2023_10_12_180159) do
+

```

Comment:
The main problem is here, in your `schema,rb` file

- [ ] Did you perform `rails generate devise:install?` because I see that you have not created a migration with the required fields

- [ ] Also make sure that you restart the application. After performing all devise setup steps for the changes to take effect

- [ ] Check out your migration file. 

```ruby
create_table "users", force: :cascade do |t|
    t.string "name"
    t.string "photo"
    t.text "bio"
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.integer "posts_counter", default: 0
  end
```

Expected 👇  

```ruby
 create_table "users", force: :cascade do |t|
    t.string "name"
    t.string "photo"
    t.string "bio"
    t.integer "posts_counter", default: 0
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.string "email", default: "", null: false
    t.string "encrypted_password", default: "", null: false
    t.string "reset_password_token"
    t.datetime "reset_password_sent_at"
    t.datetime "remember_created_at"
    ...
end
```

Reply:
![image](https://github.com/henask12/Blogify/assets/37038066/15d7b07d-cda2-4182-80c2-38d68e6a81a5)